src/helpers/application_helper.cr

```diff
@@ -4,4 +4,12 @@ module ApplicationHelper
   @container_class = "container"
   @meta = "Amber aims to deliver developer happiness, productivity and bare metal performance"
   @development : Bool? = Amber.env.development?
+
+  # vMajor.minor.path - mm/dd/yyyy (v1.4.1 - 11/8/2022)
```

Comment:
We can use releases to automate keeping the site up to date: https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28

This may be worth doing using some JS instead of run-time since it would require a query to Github. I'd like to keep the page loading as quickly as possible